src/olympia/reviewers/tests/test_views.py

```diff
@@ -3400,6 +3400,36 @@ def test_compare_link_auto_approved_but_confirmed_not_ignored(self):
         ]
         check_links(expected, links, verify=False)
 
+    def test_compare_link_not_auto_approved_but_confirmed(self):
+        first_file = self.addon.current_version.files.all()[0]
+        first_file.update(status=amo.STATUS_PUBLIC)
+        self.addon.current_version.update(created=self.days_ago(3))
+
+        confirmed_version = version_factory(addon=self.addon, version='0.2')
+        confirmed_version.update(created=self.days_ago(2))
+        confirmed_file = confirmed_version.files.all()[0]
+        AutoApprovalSummary.objects.create(
+            verdict=amo.NOT_AUTO_APPROVED, version=confirmed_version,
+            confirmed=True)
```

Comment:
Remove `confirmed=True`, this should not be set since it was not auto-approved.

Reply:
Done